### DISCUSSION TITLE
[Feature]: Log more informations

### DISCUSSION THREAD

**Discussion by l-margiela** [immich-app/immich#1631] [Feature Request]
### Feature detail

Recently, I had an issue where I couldn't upload files via the mobile app. The resolution was to increase file upload limits on Immich's ingress. That problem could have been identified sooner if the mobile app logged HTTP error 413 that it (probably) was receiving.

I can imagine this won't be the only issue that could be debugged faster with richer logs.

I propose an option to set log level in the mobile app that'd allow the user to choose how much information they need at the time. There should be at least two options, named e.g. `info` and `verbose`, with latter logging everything the app can.

### Platform

Mobile App

**Comment by samip5** [comment]
This would be useful when login doesn't seem to work without any error on client too and log shows nothing on mobile.

**Comment by nubpro** [comment]
Any ideas how can I raise this limit on Synology?

**Reply by nubpro** [reply]
I think I got it following this tutorial [/etc/nginx/sites-enabled/server.ReverseProxy.conf](https://www.synoforum.com/resources/synology-reverse-proxy-under-the-hood.135/)

1. ssh into nas
2. login with your username
3. `sudo -i`
4. `cd /etc/nginx/conf.d`
5. `vi http.custom.conf` feel free to change the file name but it must be `http.*.conf`
6. Write this, then press `Esc` then write `:wq` then `Enter`.
```
server {
    server_name <replace this with your domain. eg: google.com> ;

    client_max_body_size 5000M;
}
```

7. `sudo nginx -t -c /etc/nginx/nginx.conf.run` for validation
8. `sudo nginx -s reload -c /etc/nginx/nginx.conf.run`
9. `sudo systemctl restart nginx`
10. `sudo systemctl status nginx`
